parm/wflow/prep.yaml

```diff
@@ -29,8 +29,12 @@ task_make_grid:
 
 task_make_orog:
   <<: *default_task
-  command: '&LOAD_MODULES_RUN_TASK; "make_orog" "&JOBSdir;/JREGIONAL_MAKE_OROG"'
+  command: !cycstr 'source &USHdir;/load_modules_wflow.sh hera ; python &SCRIPTSdir;/make_orog.py
```

Comment:
I'd be fine either way. I don't personally have a strong opinion on this one, but I would be happy to adopt one if there's an obvious benefit of one over the other that I haven't picked up on yet.

Reply:
I prefer the `#!` mechanism because invocations are more concise. And I think it's more idiomatic: Although you _can_ run a bash script as `bash a.sh`, this is seldom seen in practice: Usually `a.sh` starts with `#!/usr/bin/env bash` (or similar), the executable bit is set, and the script is invoked as `/path/to/a.sh`.

Better yet, the extension can left off, since the `#!` line defines the interpreter to be used on the file, and editors (emacs and vim at least) honor the `#!` line to set the language type when editing. If these conventions were followed in general then, for example, a Bash script `myscript` could be replaced at any time with a Python script `myscript` and no calling code would have to change. The point of these conventions is to make the implementation details of "programs" invisible: You just run the program by name and it doesn't matter if it's a script in some interpreted language, or a compiled-code executable.

But if any of this is too much for here/now, that's fine.

Reply:
I appreciate the well-formed argument for the "why" and am happy to make these changes both here and for the UPP script that already exists.

Reply:
It could be a demonstration of a helpful practice.

Reply:
I'm sure you already know, but `git` will track the executable bits as a property of a revision-controlled file, so if you `chmod +x` the file, `git` will see that and allow you to commit that as a change so that the bits will be set when the files are checked out in a clone.